ide/db.dataview/nbproject/project.properties

```diff
@@ -26,7 +26,7 @@ test.unit.cp.extra=\
         ${basedir}/external/ooxml-schemas-1.4.jar:\
         ${basedir}/external/xmlbeans-3.1.0.jar:\
         ${basedir}/external/commons-collections4-4.4.jar:\
-        ${basedir}/external/commons-compress-1.20.jar
+        ${nb_all}/ide/libs.commons_compress/external/commons-compress-1.21.jar
```

Comment:
How is commons-compress made available to `db.dataview` without a dependency? Even if it is not strictly necessary, it would be good to have it, so scan of user of the module will also find it.

Reply:
Better to have it.